### DISCUSSION TITLE
Mouse cursor disappears on window title bars and more.

### DISCUSSION THREAD

**Discussion by userse31** [lxqt/lxqt#2475] [Q&A]
##### Expected Behavior
The mouse cursor DOESN'T disappear.                            

##### Current Behavior
When mousing over certain menu popups, or any window title bar, the mouse cursor disappears.

##### Steps to Reproduce (for bugs)

Evidence: https://youtu.be/0ARXAGf6oEM

It might be related to closing the lid and the laptop going to sleep?

##### Context

It's irritating.

##### System Information

* Distribution & Version: Debian 12.1
* Kernel: 6.1.0-11-amd64
* Qt Version: 1.2.0
* liblxqt Version: 1.2.0
* Package version: 1.2.0

**Comment by tsujan** [comment]
LXQt doesn't draw the cursor and can't create visual artifacts.

Your Youtube video shows that your problem is about graphics. You could start by checking the settings of your window manager and/or compositor.

**Reply by userse31** [reply]
It turned out to be python and the weird "import-im6.q16: unable to grab mouse"... thing?

(Especially odd as said python script did that, or just exited immediately under python2.7. What a WEIRD bug.)

Now, where is that "close issue" button?
